extra-i18n/fcitx5/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=fcitx-5
+PKGDES="maybe a new fcitx."
+PKGDEP="enchant iso-codes cairo xkeyboard-config libxkbcommon pango systemd wayland wayland-protocols xcb-imdkit xcb-util-wm fmt ruby-gdk-pixbuf2"
```

Comment:
```suggestion
PKGDEP="enchant iso-codes cairo xkeyboard-config libxkbcommon pango systemd wayland wayland-protocols xcb-imdkit xcb-util-wm ruby-gdk-pixbuf2"
```

Please move `fmt` to `BUILDDEP`. Since it is a [static library](https://en.wikipedia.org/wiki/Static_library#Linking_and_loading), it is not needed at the runtime.